app/components/playfair/Scrollytell.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import Column from "../layout/Column";
+import Recreation from "./Recreation";
+import Content from "./scroll_elements/Content";
```

Comment:
One other nit, `scroll_elements` should be `scrollElements`. Also should this be nested inside `playfair`? Or can we use this more generally?

Reply:
It should be general enough to be shared. That should at least be a goal. Fair nit on the naming.